lib/pure/times.nim

```diff
@@ -325,26 +548,18 @@ proc `-`*(ti1, ti2: TimeInterval): TimeInterval =
   ## Time components are compared one-by-one, see output:
   ##
   ## .. code-block:: nim
-  ##     let a = fromSeconds(1_000_000_000)
-  ##     let b = fromSeconds(1_500_000_000)
+  ##     let a = fromUnix(1_000_000_000)
+  ##     let b = fromUnix(1_500_000_000)
   ##     echo b.toTimeInterval - a.toTimeInterval
   ##     # (milliseconds: 0, seconds: -40, minutes: -6, hours: 1, days: -2, months: -2, years: 16)
```

Comment:
The result given here is not one of the probably infinite possibilities to express this interval. 
The following should be true, but it is not:
a + (b.toTimeInterval - a.toTimeInterval) == b
b - (b.toTimeInterval - a.toTimeInterval) == a
demo:
```nim
import times
let a = fromUnix(1_000_000_000)
let b = fromUnix(1_500_000_000)
let ti = b.toTimeInterval -  a.toTimeInterval
echo ti
echo a, " ", b
echo a, " ", a + ti, " ", a + ti == b
echo b - ti, " ", b, " ", b - ti == a
```
gives:
> (milliseconds: 0, seconds: -40, minutes: -6, hours: 1, days: -2, months: -2, years: 16)
> 2001-09-09T01:46:40+00:00 2017-07-14T02:40:00+00:00
> 2001-09-09T01:46:40+00:00 2017-07-07T02:40:00+00:00 false
> 2001-09-15T01:46:40+00:00 2017-07-14T02:40:00+00:00 false

The value for days should be: 5 not -2

You did adapt the doc comment to the new version. If you don't want to fix the relevant code, it would be probably better to remove this example or to add a remark that the result is actually not correct.